express-notes-api/server.js

```diff
@@ -0,0 +1,133 @@
+import { readFile, writeFile } from 'node:fs/promises';
+import express from 'express';
+
+const port = 8080;
+const app = express();
+app.use(express.json());
+
+const read = async () => {
+  try {
+    const data = await readFile('data.json', 'utf8');
+    const parsedData = JSON.parse(data);
+    return parsedData;
+  } catch (err) {
+    console.error(err);
+    process.exit(1);
```

Comment:
You shouldn't exit from an express server since that will mean it can no longer process requests. Instead, return a 500 to the caller.

The easiest way to do that is to _not_ use try/catch around the `readFile` here and just let the exception get caught in the code that calls `read()`, which will be wrapped in a try/catch.